styles/styles.css

```diff
@@ -206,48 +207,89 @@ header {
 h1, h2, h3,
 h4, h5, h6 {
   font-family: var(--heading-font-family);
-  font-weight: var(--font-weight-bolder);
+  font-weight: var(--font-weight-bold);
   line-height: 1.25;
-  margin-top: 1em;
-  margin-bottom: .5em;
   scroll-margin: calc(var(--nav-height) + 1em);
 }
 
-h1 { font-size: var(--heading-font-size-xxl) }
-h2 { font-size: var(--heading-font-size-xl) }
-h3 { font-size: var(--heading-font-size-l) }
-h4 { font-size: var(--heading-font-size-m) }
-h5 { font-size: var(--heading-font-size-s) }
-h6 { font-size: var(--heading-font-size-xs) }
+h1 {
+  font-size: 34px;
+  line-height: 1.21;
+  font-weight: var(--font-weight-boldest);
+  margin-bottom: 25px;
+}
+
+h2 {
+  font-size: var(--heading-font-size-l);
+  line-height: 1.2;
+  margin-bottom: 15px;
+}
+
+h3 {
+  font-size: var(--heading-font-size-s);
+  line-height: 1.15;
+  margin-bottom: 15px;
+}
+
+h4 {
+  font-size: var(--heading-font-size-xs);
+  line-height: 1.25;
+  margin-bottom: 9px;
+}
+
+h5 {
+  font-size: 16px;
```

Comment:
I would add a variable for the font-size `--heading-font-size-xxs: 16px`

Reply:
there are too many missing values, intermixing with existing ones. Collect all of them and rename the existing one is too big to undertake in this PR.

Reply:
ok